tests/Feature/BookTest.php

```diff
@@ -48,6 +49,12 @@ public function test_get_all_books_success(): void
             });
     }
 
+    public function test_no_books_found(): void
+    {
+        $response = $this->getJson('api/books/');
+        $response->assertStatus(404);
+    }
+
```

Comment:
Check that message is as expected, so it can't be changed from what it should be

Reply:
done